.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    "env": {
+        "commonjs": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly",
+        "document": "readonly",
+        "navigator": "readonly",
+        "window": "readonly",
+        "$": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+    // define the rules to override default rules list
+    "rules": {
```

Comment:
But why it's empty?

Reply:
Basic rules are defined in `"extends": "eslint:recommended",`

Reply:
Fixed.